extensions/wikia/Email/templates/src/forgotPassword.mustache

```diff
@@ -0,0 +1,16 @@
+<link rel="stylesheet" href="../../styles/forgotPassword.css" />
+
+<style>
+	p {
+		text-decoration: underline;
+		font-weight: bold;
+	}
```

Comment:
These are just dummy styles for now to show that we can use `style` tags as well as external style sheets. I can remove it though since we shouldn't ever be adding styles this way.